app/controllers/location_bookstores_controller.rb

```diff
@@ -20,5 +20,4 @@ def create
     bookstore.save
```

Comment:
Because you are calling the create method here, you should not need to to save since `create` is doing both a `new` and a `save`

Reply:
great catch!